docs/decisions/0016-event-design-practices.rst

```diff
@@ -0,0 +1,127 @@
+16. Event Design Practices
```

Comment:
Nit: => Event Design Best Practices? Were you intentionally avoiding calling these "best practices"?